sdk/healthdataaiservices/azure-resourcemanager-healthdataaiservices/CHANGELOG.md

```diff
@@ -1,14 +1,12 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0 (2024-11-15)
```

Comment:
release date

Reply:
@XiaofeiCao 
Change the release date to today?

Reply:
yes

Generally, you should close the PR if this is only for test whether there is breaking changes. Run a new one when you need release.